tinygrad/renderer/triton.py

```diff
@@ -58,14 +58,14 @@ def kk(s): kernel.append("  "*depth+s)
     UnaryOps.SQRT: lambda x,: f"tl.sqrt({x})",
     UnaryOps.NEG: lambda x,: f"-{x}",
     BinaryOps.ADD: lambda x,y,: f"({x}+{y})", BinaryOps.SUB: lambda x,y,: f"({x}-{y})",
-    BinaryOps.MUL: lambda x,y,: f"({x}*{y})", BinaryOps.DIV: lambda x,y,: f"({x}/{y})",
+    BinaryOps.MUL: lambda x,y,: f"({x}*{y})", BinaryOps.DIV: lambda x,y,: f"({x}/{y})" if y != '0.0' else f"{x}*tl.where({x}==0.0, float('nan'), float('inf'))",
```

Comment:
I don't like that these compares are to a string

Reply:
`x` can be a variable name so we can't cast to float and since triton can handle
``` 
x = 0
y = 1/x
```
and cant handle `y=0/0` it is enough to just compare strings. I could generate the comparison in code but I think that is unnecessary computation that can be resolved during codegen